resources/carrier/en/98.txt

```diff
@@ -21,10 +21,18 @@
 98931|MTCE
 98932|Taliya
 98933|Irancell
-98934|TKC
+98934|TeleKish
 98935|Irancell
 98936|Irancell
 98937|Irancell
 98938|Irancell
 98939|Irancell
 98990|IR-MCI
+98991|IR-MCI
+9899810|Shatel Mobile(Full MVNO)
+989944|Ansarestan(Light MVNO)
+9899900|LotusTel(Full MVNO)
+9899911|ApTel(Light MVNO)
```

Comment:
It's because if you look at Wikipedia page, really 9991(ApTel) and Azartel(99914) are two brands of one company(Named Negin Tel). Negin Tel gave 9991 from CRA(Regulatory) and divided it to two brands (Aptel=9991) and Azartel(99914). It means they forked Azartel (With 99914) from their Main brand(Aptel=9991).
I**f you look at numbering plan file(From ITU), you will not see 99914 while you can see it on Azartel website! It's because it's internal branding in Negin Tel.** (Website: http://www.negintel.com/index.html)
It's because Azartel branding focused on only three provinces in Iran(East Azerbaijan Province, Azerbaijan Province and Ardabil Province).
Really i should change it to like it:
99911=ApTel
99912=ApTel
99913=ApTel
99914=Azartel
99915=ApTel
99916=ApTel
99917=ApTel
99918=ApTel
99919=ApTel
But for now, i only added 99911 only from Aptel and 99914 from Azartel. Because Azartel Started working with 99914 and ApTel currently registering for 99911 only. Altough all of 9991(Based on ITU Docs) and other news are for NeginTel. But I'm not sure about future status of ther phone numbers. For example, if Azartel brand developed, maybe company(NeginTel) want to give 99914 to 99919 to Azartel and 99911 to 99913 to ApTel or even create new brands for other numbers. If any changes happened=I will add that changes to libphonenumber.

It's not possible that we slect NeginTel for all of 9991 numbers. Because users and poeple know them as ApTel and Azartel. They don't know NeginTel.

Evidince=Azartel and Aptel websites and check their number range. In addition, you can check wikipedia and Numbering Plan(My Numbering Plan).
In addition, it's NeginTel website that verifying really NeginTel is MVNO. But NeginTel's Brands are ApTel and Azartel
Pay attention: In iran, because companies law, in most of cases, companies are working with their brands not with their names!

Reply:
Going ahead with range details mentioned in ITU doc sounds good to me.